src/main/java/com/hbm/tileentity/machine/TileEntityCustomMachine.java

```diff
@@ -160,12 +165,12 @@ public void updateEntity() {
 					this.tryProvide(worldObj, pos.getX(), pos.getY(), pos.getZ(), pos.getDir());
 				for (FluidTank tank : this.outputTanks)
 					if (tank.getFill() > 0)
-						this.sendFluid(tank, worldObj, pos.getX(), pos.getY(), pos.getZ(), pos.getDir());
+						this.tryProvide(tank, worldObj, pos.getX(), pos.getY(), pos.getZ(), pos.getDir());
 				this.sendSmoke(pos.getX(), pos.getY(), pos.getZ(), pos.getDir());
 			}
 
 			if (this.structureOK) {
-
+				float volume = 1F;
```

Comment:
This is still serverside so it does nothing. Volume is never anything other than 1 here.